### ISSUE TITLE
Vehicle does not move after completely stopped 

### ISSUE THREAD

**Issue by RoyY-S** [AriGlockner/RacingGame#2]
**Description of the bug:**
The vehicle? car? when completely stop, it never go again by pressing W, but pressing A or D can make the wheels turn; seems the only way to make the car moving again, is to press S let the car reverse first, then press W will works which move forward and accelerate.

**How do I reproduce this bug?**
1. Press W to let the vehicle moving for a bit
2. Long press Space to make it stop completely/idle
3. Press W or up-arrow again, the vehicle does not go

**What should be happening:**
The vehicle should move forward directly by just pressing W or up-arrow like before/when it's moving

**Screenshot/clip show the problem:**

https://user-images.githubusercontent.com/78105985/117727319-b97e2a80-b19c-11eb-9aa2-8842e61c033a.mp4

_The top-center of video showing the key I'm pressing_
So basically I had to make the vehicle move backward a bit before it can go forward.

**Comment by AriGlockner** [comment]
I didn't realize that the space key was an in game button. I will look through this more on Thursday. I might fix this, or I may just remove the space key. Thanks for catching this bug. I will leave this case open while I look at it.

**Comment by RoyY-S** [comment]
> I didn't realize that the space key was an in game button. I will look through this more on Thursday. I might fix this, or I may just remove the space key. Thanks for catching this bug. I will leave this case open while I look at it.

I think there might be another option rather than just get rid of that button since it is pretty useful: telling/instruct players that the vehicle requires some sort of "activation" before it can go, assuming that is a normal mechanism/characteristic of car.

**Comment by AriGlockner** [comment]
I'm not going to worry about the breaking in this game. Until you mentioned that breaking was a thing in this game, I didn't know it was included. I have bigger priorities than to fix this considering the car movement wasn't something I wrote the code for. Keep finding bugs and submitting them. I will likely release the next version of this game soon.
